src/Microsoft.Health.Dicom.Api/Registration/DicomServerApplicationBuilderExtensions.cs

```diff
@@ -56,6 +72,39 @@ public static IApplicationBuilder UseDicomServer(this IApplicationBuilder app)
             app.UseStaticFiles();
         }
 
+        app.UseRouting();
+        useDevelopmentIdentityProvider?.Invoke(app);
+        useHttpLoggingMiddleware?.Invoke(app);
+
+        app.UseEndpoints(endpoints =>
+        {
+            endpoints.MapControllers();
+            endpoints.MapHealthChecks(KnownRoutes.HealthCheck,
```

Comment:
Is `MapHealthChecks` necessary if `UseCachedHealthChecks` is above in the pipeline?

Reply:
Removed `MapHealthChecks` as discussed offline.